### PR TITLE
mpc85xx: Improve LED support for TP-Link TL-WDR4900 V1

### PR DESCRIPTION
**Pull request by CHKDSK88**
This PR improves LED support in TP-Link TL-WDR4900 V1.

It introduces usb trigger support and adds LED nodes to dts.

### CODE REVIEW

**Comment by xabolcs** [comment]
There is a typo in the usb trigger commit description:
> Moderns kernel can handle usb
led trigger.

"Modern kernels ..."

File: target/linux/mpc85xx/files/arch/powerpc/boot/dts/tl-wdr4900-v1.dts [hunk 1]
```diff
@@ -118,6 +118,23 @@
...
 			phy_type = "utmi";
 			dr_mode = "host";
+
+			port@1 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				reg = <1>;
+				#trigger-source-cells = <0>;
+
+				hub_port1: port@1 {
```
Comment (neheb) [inline target/linux/mpc85xx/files/arch/powerpc/boot/dts/tl-wdr4900-v1.dts:128]: normally regs start at 0. Hopefully this is OK.

Comment (CHKDSK88) [reply target/linux/mpc85xx/files/arch/powerpc/boot/dts/tl-wdr4900-v1.dts:128]: Yes, both leds work.

**Comment by CHKDSK88** [comment]
> There is a typo in the usb trigger commit description:
> 
> > Moderns kernel can handle usb
> > led trigger.
> 
> "Modern kernels ..."

Thanks. Fixed.

**Review by neheb — APPROVED** [review]

**Comment by robimarko** [comment]
Thanks! Rebased on top of main and merged!

**Pull request merged**